blocks/identity-block/intl.json

```diff
@@ -1179,6 +1226,7 @@
 		"bn": "**********",
 		"bo": "**********",
 		"de": "**********",
+		"el": "**********",
```

Comment:
curious! Why do we need translation scripts for * s and it looks same in all languages

Reply:
We don't really, I'm not sure why this was added. It would be a question for the subs team if there was a plan to put something else here eventually.

Reply:
I am passing it for now.